samples/bluetooth/advertiser/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Nothing wrong on this line, but the commit message is just this:
>  Adds two more Bluetooth samples
>
> Signed-off-by: Yeshvanth M <yeshvanthmuniraj@gmail.com>

That is simply insufficient for a commit message, especially if looking at this during a bisect.

Please write some more info, for example:
- What sample is added
- What can this sample do
- Anything else that is special or valuable to know about this sample

Please note, the commit message is also an aid to the reviewers in order to understand the code and why it is being added.

Reply:
I've added the required details to the commit message.